examples/hybrid/cli_options.jl

```diff
@@ -191,6 +191,10 @@ function parse_commandline()
         help = "Define the surface elevation profile [`NoWarp`,`Earth`,`DCMIP200`]"
         arg_type = String
         default = "NoWarp"
+        "--dt_save_restart"
+        help = "Time between saving restart files to disk. Examples: [`10secs`, `1hours`, `Inf` (do not save)]"
+        arg_type = String
+        default = "Inf"
```

Comment:
Isn't this what `dt_save_to_disk` is doing?

Reply:
Differences are:
1. `dt_save_to_disk` collects data from all processors into root and dump into one single data file -> making it easy for later on remapping and post-processing; while `dt_save_restart` saves data into separate files from there own processor -> so that restart of mpi can be initiated by reading from the separate files;
2. `dt_save_to_disk` saves the state `Y` plus a lot of diagnostics while `dt_save_restart` saves `Y` only;
3. usually we want different frequencies in saving data for the post-processing purpose vs the restart purpose.

Reply:
Ah, ok, thanks, makes sense. Should we combine this PR and #672?